lib/identity/hostdata/ec2.rb

```diff
@@ -21,6 +21,12 @@ def self.http
         http
       end
 
+      # get token
+      def self.v2_token
+        token = Net::HTTP.new('169.254.169.254', 80)
+        http.put('/latest/api/token', nil, { 'X-aws-ec2-metadata-token-ttl-seconds': "60" }).body
```

Comment:
1. do we need the `token` variable? We can just use the `http` method right?
2. let's use a consistent quote style and drop the optional braces
3. just in case there's a newline or something in the body, let's throw a `#chomp` on there too
```suggestion
        http.put('/latest/api/token', nil, 'X-aws-ec2-metadata-token-ttl-seconds' => '60').body.chomp
```